charts/posthog/values.yaml

```diff
@@ -581,7 +597,9 @@ pgbouncer:
     - name: PGBOUNCER_MAX_CLIENT_CONN
       value: "1000"
     - name: PGBOUNCER_POOL_MODE
-      value: "transaction"
+      value: transaction
+    - name: PGBOUNCER_IGNORE_STARTUP_PARAMETERS
+      value: extra_float_digits
```

Comment:
I've added this as the exporter requires it. Looking at our codebase I don't think we rely on this setting anywhere.